docs/reference/esql/esql-apis.asciidoc

```diff
@@ -0,0 +1,20 @@
+[[esql-apis]]
+== {esql} APIs
+
+The {es} Query Language ({esql}) provides a powerful way to filter, transform,
+and analyze data stored in {es}, and in the future in other runtimes. For an
+overview of {esql} and related tutorials, see <<esql>>.
+
+* <<esql-query-api>>
```

Comment:
In the docs preview, this link is rendered in italics, not sure why.

Reply:
Seems to be caused by different levels of discrete heading in the doc titles. I updated the query-api doc to match that of others, and this seems to have resolved it.